express-delete/index.js

```diff
@@ -0,0 +1,45 @@
+const express = require('express');
+const app = express();
+
+app.listen(3000, () => {
+});
+
+const grades =
+{
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92
+  }
+};
+
+app.get('/api/grades', (req, res) => {
+  const gradesArr = [];
+  for (const entry in grades) {
+    gradesArr.push(grades[entry]);
+  }
+  res.json(gradesArr);
+});
+
+app.delete('/api/grades/:id', (req, res) => {
+  delete grades[req.params.id];
+  const gradesArr = [];
+  for (const entry in grades) {
+    gradesArr.push(grades[entry]);
+  }
+  res.json(gradesArr);
```

Comment:
You can remove lines 39-43. This route is only supposed to delete the grade from the data model and send a `204` response.